src/main/java/org/broadinstitute/hellbender/tools/spark/transforms/markduplicates/MarkDuplicatesSpark.java

```diff
@@ -106,6 +107,8 @@
 
         // Here we combine the original bam with the repartitioned unmarked readnames to produce our marked reads
         return sortedReadsForMarking.zipPartitions(repartitionedReadNames, (readsIter, readNamesIter)  -> {
+            System.out.println("system property says: " + System.getProperty("samjdk.compression_level"));
+            System.out.println("htsjdk says: " + Defaults.COMPRESSION_LEVEL);
```

Comment:
@jamesemery These debug print statements seem to have snuck into your latest commit -- can you remove them?

Reply:
whoops, yeah they are gone